docs/MIGRATING_V1.md

```diff
@@ -0,0 +1,316 @@
+# Upgrading to Project Factory v1.0
```

Comment:
Per our style guide, the file name and the title should match. I think the file name can be more verbose.

Reply:
Renamed to `docs/upgrading_to_project_factory_v1.0.md`